GOT-StudentVersion/GOT-StudentVersion/GOTEpisdoe.swift

```diff
@@ -98,4 +110,56 @@ class GOTEpisode {
         GOTEpisode(airdate: "2017-08-20", id: 1221414, name: "Beyond the Wall", number: 6, season: 7, runtime: 60, summary: "Jon's mission continues north of the wall, but the odds against his ragged band of misfits may be greater than he imagined.", mediumImageID: "312651", originalImageID: "312651"),
         GOTEpisode(airdate: "2017-08-27", id: 1221415, name: "The Dragon and the Wolf", number: 7, season: 7, runtime: 60, summary: "Cersei sits on the Iron Throne; Daenerys sails across the Narrow Sea; Jon Snow is King in the North, and winter is finally here.", mediumImageID: "314502", originalImageID: "314502")
     ]
+    
+    
+    private static let seasonOne = GOTEpisode.allEpisodes.filter({$0.season == 1})
+    
+    private static let seasonTwo = GOTEpisode.allEpisodes.filter({$0.season == 2})
+    
+    private static let seasonThree = GOTEpisode.allEpisodes.filter({$0.season == 3})
+    
+    private static let seasonFour = GOTEpisode.allEpisodes.filter({$0.season == 4})
+    
+    private static let seasonFive = GOTEpisode.allEpisodes.filter({$0.season == 5})
+    
+    private static let seasonSix = GOTEpisode.allEpisodes.filter({$0.season == 6})
+    
+    private static let seasonSeven = GOTEpisode.allEpisodes.filter({$0.season == 7})
+    
+    static var nameFilters = ["": getAllSeasons()]
+    static var summaryFilters = ["": getAllSeasons()]
+    
+    static let allSeasons = getAllSeasons()
+
+    
+    private static func getAllSeasons() -> [[GOTEpisode]] {
+        return [ GOTEpisode.seasonOne,
+                 GOTEpisode.seasonTwo,
+                 GOTEpisode.seasonThree,
+                 GOTEpisode.seasonFour,
+                 GOTEpisode.seasonFive,
+                 GOTEpisode.seasonSix,
+                 GOTEpisode.seasonSeven
+                ]
+    }
+    
+    static func getFilteredResults(selectedStr: String, searchText: String) -> [[GOTEpisode]] {
```

Comment:
Great job handling this functionality in the model! It makes sure your VC is clean and not filled with this search bar logic